tutorials/algorithms/11_VarQTE.ipynb

```diff
@@ -0,0 +1,911 @@
+          },
+          "metadata": {}
+        }
+      ],
+      "source": [
+        "import pylab\n",
+        "\n",
+        "H_exp_val = np.array([ele[0][0] for ele in evolution_result.observables])\n",
```

Comment:
I have explicitly added the suggestion here, but I see there are other occurences later on. Could you please update all variables that start with an uppercase letter to use lowercase?

```suggestion
        "h_exp_val = np.array([ele[0][0] for ele in evolution_result.observables])\n",
```